### ISSUE TITLE
Minimum height

### ISSUE THREAD

**Issue by Mo8it** [lakoliu/Furtherance#16]
The current minimum height is not helpful for showing useful information.

The welcome window is also awful with the current minimum height.

The minimum height should allow showing at least two tasks, in my opinion.

# Current minimum height
![image](https://user-images.githubusercontent.com/76752051/161746088-b0e3ffe8-7616-455c-b0d4-76d1e1093408.png)

# Recommended minimum height
![image](https://user-images.githubusercontent.com/76752051/161746140-677cbbff-27e8-43a6-abd7-285d5b07c1e3.png)

**Comment by lakoliu** [comment]
I will take this into consideration. In my experience, people get a bit frustrated when you don't allow them to make the app whatever size they want. But I'll leave this open for now and think it over. If anyone else has an opinion I'd love to hear it.

**Comment by lakoliu** [comment]
It was slightly difficult to pick a minimum window height and get this issue fixed, but I did indeed set one. The issue with your recommended height was that it showed an odd portion of the next day's heading if there were multiple days of tasks in the list and only one task on the current day. Therefore, to make two tasks show without any odd behavior it would look like this: 
![Screenshot from 2022-04-08 11-54-07](https://user-images.githubusercontent.com/99976966/162403413-07d69d22-0a41-460a-8b44-e2415c5d9a87.png)
I decided to go instead with the minimum height showing only one task to remove any strange behavior. Therefore, the next release will look like this when the user drags the window to the minimum height:
![Screenshot from 2022-04-08 12-02-53](https://user-images.githubusercontent.com/99976966/162403606-798ab726-8543-407d-9875-998dddf12a2a.png)
